aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -133,6 +133,122 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 				},
 			},
 
+			"extended_s3_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
```

Comment:
We should add a `ConflictsWith` for the `s3_configuration` attribute, agreed? 

- https://github.com/hashicorp/terraform/blob/master/helper/schema/schema.go#L163

ex: 

- https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_alb.go#L49